cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -20,19 +19,22 @@
 /**
  * This class realizes the functionality that is needed for the create commands.
  */
-@CommandLine.Command(name = "create", header = "%n@|green Create-Controller|@", subcommands = {
-    CreateController.CreateServiceCommand.class,
-    CreateController.AssignSpaceDeveloperCommand.class,
-    CreateController.CreateApplicationCommand.class })
+@CommandLine.Command(name = "create", header = "%n@|green Create a new app, service or add a new " +
+        "space developer.|@",
+        mixinStandardHelpOptions = true,
+        subcommands = {
+                CreateController.CreateServiceCommand.class,
+                CreateController.AssignSpaceDeveloperCommand.class,
+                CreateController.CreateApplicationCommand.class})
 public class CreateController implements Runnable {
 
     @Override
     public void run() {
-        // this code is executed if the user runs the create command without specifying
-        // any sub-command
+        CommandLine.usage(this, System.out);
```

Comment:
Is the name of that call defined by us? If yes, I'd suggest `displayUsage`, which is more descriptive.

Reply:
No it is from the picocli library.

Reply:
I see, thanks.